src/core/options.cpp

```diff
@@ -75,6 +75,7 @@ bool ENABLE_TYPE_FEEDBACK = 1 && _GLOBAL_ENABLE;
 bool ENABLE_RUNTIME_ICS = 1 && _GLOBAL_ENABLE;
 bool ENABLE_JIT_OBJECT_CACHE = 1 && _GLOBAL_ENABLE;
 
+// TODO(rntz): do we ever not want to have frame introspection? should we remove this option?
```

Comment:
agreed that it's pretty rare that we flip this off.  I think we test it every now and then to see what the perf impact is of adding all the extra LLVM args.